### DISCUSSION TITLE
Minifier

### DISCUSSION THREAD

**Discussion by trymeouteh** [rolldown/rolldown#1018] [Ideas]
Please add the ability to build minified builds and non minified builds. For example...

- app.js
- app.min.js

**Comment by trymeouteh** [comment]
I did find this Go project that is a minifier for JS and CSS

https://github.com/tdewolff/minify

**Comment by hyfdev** [comment]
Yeah. A builtin minifier will be supported, which is actually oxc-minifer under the hood.

**Comment by trymeouteh** [comment]
I also found this JS minifier written in Rust

https://github.com/wilsonzlin/minify-js
